Exiled.API/Features/Roles/HumanRole.cs

```diff
@@ -28,6 +28,6 @@ internal HumanRole(Player player, RoleType type)
         public override Player Owner { get; }
 
         /// <inheritdoc/>
-        public override RoleType Type { get; }
+        internal override RoleType RoleType { get; }
```

Comment:
Make it field instead.

Reply:
Fields cannot be abstract

Reply:
Bruh, how do you think properties work in C#?
Anyway, don't make it abstract, where's the problem?